nx.json

```diff
@@ -20,9 +20,7 @@
       }
     }
   },
-  "workspaceLayout": {
-    "libsDir": "packages"
-  },
+  "workspaceLayout": { "appsDir": "apps", "libsDir": "libs" },
```

Comment:
We shouldn't commit this, the usage of libs is only for the documentation website.